DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/ToSpeckleScalingService.cs

```diff
@@ -0,0 +1,29 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.RevitShared.Helpers;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public sealed class ToSpeckleScalingService
+{
+  private readonly double _defaultLengthConversionFactor;
```

Comment:
How is this being used. I wonder if this scaling factor should be part of the conversion context stack?

Reply:
this is being used for speed. I once profiled a very large revit model and found that we were computing this conversion factor over 5 million times. So now we store that value and use it in our default "ScaleToSpeckle" method.

Yeah I should maybe talk to Alan about that. It seems like it kind of overlaps with the unit functionality of the conversion stack.

Reply:
Yep this sounds like a context thing to me :) lets chat later!

Reply:
Although creating and disposing of elements in short order 5 millions times might be bad. Was it a necessary conversion, or just that we were doing it in many unnecessary places?